src/imitation/scripts/config/data_collect.py

```diff
@@ -100,6 +103,33 @@ def walker():
   env_name = "Walker2d-v2"
 
 
+# Custom env configs
+
+@data_collect_ex.named_config
+def two_d_maze():
+  env_name = "imitation/TwoDMaze-v0"
+
+
+@data_collect_ex.named_config
+def custom_ant():
+  env_name = "imitation/CustomAnt-v0"
+  make_blank_policy_kwargs = dict(
```

Comment:
Do we envisage a situation where `ant`, `custom_ant` and `disabled_ant` would have different hyperparameters? If not, can we factor out into a helper function/dictionary? Sadly might need to be a bit hacky, along the lines of `locals().update(...)`

Reply:
Not sure if `locals().update(...)` is okay. (Maybe sacred does some magic that makes this work.)

https://docs.python.org/3/library/functions.html#locals

I'll give it a quick shot though

Reply:
It works!